lib/errors.js

```diff
@@ -51,6 +55,23 @@ Errors.prototype = {
     },
 
     /**
+     * Adds style error to the list
+     *
+     * @param {Object} errorInfo
+     */
+    cast: function(errorInfo) {
```

Comment:
Not seeing the need for this additional public function. Why not put the checks in `_addError`?

If you want to remove that logic to avoid cluttering `_addError`, then maybe put it in a `_validateError` function called within `_addError`